src/playwrightTest.ts

```diff
@@ -203,7 +210,7 @@ export class PlaywrightTest {
       type: 'pwa-node',
       name: debugSessionName,
       request: 'launch',
-      cwd: configFolder,
+      cwd: config.workspaceFolder,
       env: {
         ...process.env,
```

Comment:
I am wondering if for all these processes that get spawned if it would make sense to also set a _special_ environment variable something similar to `IS_PLAYWRIGHT=1` so people could change behaviours when code is running inside Playwright context

Reply:
We could, but I am not sure we want to encourage that behavior. Do you have a good example of the conditional behavior?

Reply:
Good point, not sure, originally I was thinking I could stop throwing error's in this situation but that wouldn't be a good way to solve things. I will try to add your suggested `onError`-listener